runtests.py

```diff
@@ -332,6 +332,12 @@ def main(argv):
     else:
         sys.exit(1)
 
+def get_path_suffix(current_path, levels=3):
```

Comment:
This could use a short docstring comment about what it's doing - easier than interpreting the code.

Reply:
So the code further down assumes that `site-packages/` is always 3 levels deep in the prefix, e.g. `lib/python3.9/site-packages`. I'm not certain that this is universally true.

Reply:
Okay, reviewing the original `get_python_lib` from `distutils/sysconfig.py`, this is indeed always the case.

And the `prefix=` option in the original `get_python_lib` was removed - it would have been quite handy to have that around still.

Reply:
> short docstring comment about what it's doing

Yeah, sorry I missed that!